code/game/machinery/camera/camera_assembly.dm

```diff
@@ -105,6 +105,8 @@
 	if(is_type_in_list(W, possible_upgrades) && !is_type_in_list(W, upgrades)) // Is a possible upgrade and isn't in the camera already.
 		if(!user.unEquip(W))
 			return
+		if(!W.drop_item()) // no eating borg items
+			return

```

Comment:
shouldn't unEquip just do those checks? This is silly we've got two ways to remove items